### ISSUE TITLE
Add profiles with future DnD integration

### ISSUE THREAD

**Issue by kamieniarz** [KartulUdus/PoracleJS#334]
Profiles is something me and users of my map are dreaming about since I remember. Basically most of us work away from home (maybe nor during covid but generally). So if someone tracks mons during work hours he needs to change area/location twice a day - when going to work and when going back home. That could be even more if someone plays in 3, 4 or more locations every day during same hours. And also that would be tricky if we're using location and want to have `d1000` at home but `d300` at work.

What I'm talking about is an option to configure profiles, that would give us an option to have 2 or more poracle sets of notifications. How would that work?

Lets say prefix `profile` is responsible for creating one:
`!profile add home` - creating profile named `home`
`!area add profilehome cityname` - profile named `home` gets an area
`!track profilehome everything iv100` - track hundos at `home`
`!profile add work` - creating profile named `work`
`!area add profilework cityname` - profile named `work` gets an area
`!track profilework everything iv100` - track hundos at `work`
So basically if there's `profilexxxx` in command it triggers saving commands in that specific profile.

Some other examples of `profile` usage:
`!profile home` - switch to `home`
`!profile remove home` - completly remove `home`
`!profile rename home home1` - change `home` to `home1`

Also with DnD https://github.com/KartulUdus/PoracleJS/issues/223 we could create schedules for each profile:
`!dnd add profilehome d1-5 h16-7` - make `home` profile work from monday to friday (`d1-5`) between 16:00 and 07:00 (`h16-7`)
`!dnd add profilehome d6-7` - make `home` profile work on weekends

I hope these feature requests are clear enough. This is trully something big that would improve tracking things a lot so I hope it'll be implemented one day ;)

**Comment by jfberry** [comment]
This is actually pretty much in line with what I was thinking... except I was just going to have numbered profiles in the first instance with no names which wouldn't require a profiles table..  eg `/track profile2 everything iv100' `/profile 2`
But they could be named also I guess...

**Comment by jfberry** [comment]
And dnd would just be another profile with switching hours...
